example/lib/main.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui';
+
+import 'package:flame/components/component.dart';
+import 'package:flame/components/mixins/resizable.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/game.dart';
+import 'package:flame/text_config.dart';
+import 'package:flame/position.dart';
+import 'package:flame/palette.dart';
+import 'package:flame_flare/flame_flare.dart';
+import 'package:flutter/material.dart';
+import 'package:flare_flutter/flare_controls.dart';
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  final game = MyGame();
+  runApp(game.widget);
+}
+
+class MyGame extends BaseGame with TapDetector, DoubleTapDetector {
+  final TextConfig fpsTextConfig = TextConfig(color: BasicPalette.white.color);
+
+  final paint = Paint()..color = const Color(0xFFE5E5E5E5);
+  final List<String> _animations = ['Wave', 'Dance'];
+  int _currentAnimation = 0;
+  MinionComponent minionComponent;
+  MinionController minionController;
+
+  bool loaded = false;
+
+  MyGame() {
+    minionController = MinionController();
```

Comment:
This instance is not used, maybe it should be instantiated inside the component class?

Reply:
Changed that.